subsys/nrf_security/src/drivers/cracen/Kconfig

```
@@ -16,7 +16,8 @@ if PSA_CRYPTO_DRIVER_CRACEN
 
 config CRACEN_LOAD_MICROCODE
 	bool "Load CRACEN microcode"
-	depends on (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX
+	depends on (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX || \
+				(SOC_SERIES_NRF92X && (SOC_NRF9280_CPUSEC || SOC_NRF9230_CPUSEC))
```

I would think that this is sufficient:
```suggestion
	depends on (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX || \
				SOC_SERIES_NRF92X
```